app/nav/CmsNavbar.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+	Flex,
+	Stack,
+	Button,
+	useColorModeValue,
+	Heading,
+} from "@chakra-ui/react";
+import { FiHome } from "react-icons/fi";
+import { Form } from "remix";
+import { url as logoutURL } from "~routes/cms.logout";
+import { url as cmsUrl } from "~routes/cms/index";
+import NavElement from "./CmsNavElement";
+
+export default function CmsNavbar({ fullName }: { fullName: string }) {
+	return (
+		<Flex
+			bg={useColorModeValue("gray.200", "gray.700")}
```

Comment:
Doesn't this violate the rule of hooks? I'd say this inline use of hooks should be avoided.

Reply:
Actually doesnt. Hooks are valid as long as they arent called in loops, conditionally or in nested functions. We can avoid this though if you'd like to